client/app/bundles/KudosApp/components/TabBar.jsx

```diff
@@ -1,29 +1,35 @@
-import React, { PropTypes } from 'react';
+import React, { PropTypes } from 'react'
+import { observer } from 'mobx-react'
+import AppStore from '../stores/AppStore'
 
-const Tab = ({ tabName, currentTab, clickHandler }) => {
-  const classNameForTab = 'nav-link' + (currentTab == tabName ? ' active' : '');
+@observer
+export class TabBar extends React.Component {
+  render() {
+    const tabs = ['Recent', 'My Kudos', 'Awarded Kudos']
```

Comment:
Let's take this opportunity to change these: "Received", "Sent"

Reply:
unfortunately this requires some rails api changes because we're gross, so i'll leave that for later!

<img width="554" alt="screen shot 2018-06-08 at 17 08 02" src="https://user-images.githubusercontent.com/17481114/41185571-93a56050-6b3e-11e8-9878-cd124e235f78.png">